pkg/consts/views.go

```diff
@@ -253,6 +254,18 @@ function(doc) {
 `,
 }
 
+// JobByQueuedAt is used to find jobs by their queued datetime. Worker type and
+// state filters also can be used for granular selection
+var JobByQueuedAt = &couchdb.View{
+	Name:    "jobs-by-queued-at",
+	Doctype: Jobs,
+	Map: `
+function (doc) {
+	emit(doc.queued_at, doc._id);
+}
+`,
+}
```

Comment:
Why do you need both a mango index on `queued_at` and a view? The mango index should be enough, and I prefer to not add views when we can use mango index.

Reply:
Views offers the `endkey` param, which is very useful to get all the jobs before a specific date with `queued_at` sorted view.

Reply:
And mango has a `$lt` keyword in `selector` to do that.